hub/api/dataset.py

```diff
@@ -344,6 +344,18 @@ def delete(self):
             self.client.delete_dataset_entry(self.org_id, self.ds_name)
             logger.info(f"Hub Dataset {self.path} successfully deleted.")
 
+    def plot(self):
```

Comment:
Don't like this. I don't think we need to implement the template if it's not going to be implemented, especially since it's unclear what it would actually do